mgmt/rest/v2/api.go

```diff
@@ -0,0 +1,74 @@
+package v2
+
+import (
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/intelsdi-x/snap/pkg/api"
+)
+
+const (
+	version = "v2"
```

Comment:
I'm not a fan of many global variables. I see the difficulty for you to change it as there are mixed methods and functions. It's ok. leave it.